src/main/resources/db/migration/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+changeLogFile: src/main/resources/db/migration/dbchangelog.xml
+url: jdbc:mysql://localhost:3306/my_database
+username: root
+password: mypassword
```

Comment:
Please reuse properties from `src/main/resources/db.properties` config file in the liquibase plugin configuration. Value of  property `changeLogFile`  can be moved to the pom.xml file becuase after refactoring this file will contain only it.